pkg/codegen/docs/templates/index_categories.tmpl

```diff
@@ -1,5 +1,5 @@
 {{- define "index_modules" }}
-<h3>Modules</h3>
+<h3 id="modules">Modules</h3>
```

Comment:
Actually now that I am look at this separately. I don't know why these don't just use `###`?

Reply:
..here and in other similar places where the text content is static. The reason we switch to the `<h*>` tag is because adding `<wbr>` tags in the title was not rendering with the markdown equivalent for the header tags.

Reply:
Yeah, these could be `###` here, but I am just going to leave them `<h3>` for now.

The `<wbr>` thing is a good catch. Will fix.

Reply:
Separate related issue I noticed. Now that these have ids, they show up on the right-hand "ON THIS PAGE"...

The package details template gets rendered both here and on individual resource/function pages, and since these are all h3, so is the package details header.

But on the individual resource/function pages, the top-level headers are all h2, which ends up making it look like "Package Details" is a subheading under "Supporting Types".

See bottom of this screenshot from the Bucket page:

<img width="280" alt="Screen Shot 2020-04-17 at 11 39 45 AM" src="https://user-images.githubusercontent.com/710598/79602869-25788700-80a0-11ea-95ba-09381c6363dc.png">

To address, I'm going to promote these h3s to h2.

Reply:
> To address, I'm going to promote these h3s to h2.

sounds good! Great that you noticed this!